tests/e2e-leg-9/new-online-upgrade-sanity/setup-vdb/base/setup-vdb.yaml

```diff
@@ -17,6 +17,7 @@ metadata:
   name: v-base-upgrade
```

Comment:
Could we have a new test "test/e2e-leg-9/multi-online-upgrade"? Then upgrade a one-subcluster db twice in that test.